blue-note/style.css

```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Blue Note
+Theme URI: https://github.com/WordPress/community-themes/
+Author: the WordPress team
+Author URI: https://wordpress.org
+Description: 
+Requires at least: 6.1
+Tested up to: 6.2
+Requires PHP: 8.0
```

Comment:
why 8.0? are we using something specific to have that high a requirement?

Reply:
No, but we should encourage users to use a newer PHP version. Consider that current version of PHP is 8.2.

Reply:
TIL: _Create block theme_ does not support PHP 8, so this should be lowered until the plugin is updated 😞

Reply:
My dev env has wp_debug enabled and with PHP 8 I get a ton of warnings that are coming from core. I think in general going with 7 for now should be fine until core updates. In any case, CBT should update as well, I believe I saw an issue raised in their repo about it